src/6model/serialization.c

```diff
@@ -10,7 +10,7 @@
 
 /* Version of the serialization format that we are currently at and lowest
  * version we support. */
-#define CURRENT_VERSION 19
+#define CURRENT_VERSION 20
```

Comment:
I don't think it will fail to deserialize properly since you didn't add any version check code, and no STABLE_HAS_HLL_ROLE flag in the old bytecode files.